server/app/app.go

```diff
@@ -155,14 +155,7 @@ func (a *App) Start() error {
 			a.config.Server.PathPrefix,
 			"./html",
 			"index.html",
-			map[string]string{
-				"AnalyticsKey":     analytics.FrontendKey,
-				"AnalyticsEnabled": fmt.Sprintf("%t", a.config.GA.Enabled),
-				"ServerPathPrefix": fmt.Sprintf("%s/", a.config.Server.PathPrefix),
-				"ServerID":         serverID,
-				"AppVersion":       Version,
-				"Env":              Env,
-			},
+			a.config.GetFrontendConfig(analytics.FrontendKey, serverID, Version, Env),
```

Comment:
Moving this to the `config.go` file

Reply:
please revert this. This not a `config` thing, but a `render` thing